### ISSUE TITLE
NTR: VDRL positive in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8444]
New term request VDRL positive in CSF (segal_041222192739): Cerebrospinal fluid  UMLS: 0241659

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000395
- label: Positive CSF VDRL test
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:32869202
- def: An abnormal (positive) result of the Venereal Disease Research Laboratory (VDRL) test performed in cerebrospinal fluid (CSF), suggesting past or present neurosyphilis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8444
- synonyms:
